lib/optimizer/configuration/can-override.js

```diff
@@ -1,19 +1,21 @@
 var understandable = require('./properties/understandable');
 
 function animationIterationCount(validator, value1, value2) {
-  if (!understandable(validator, value1, value2, 0, true) && !(validator.isAnimationIterationCountKeyword(value2) || validator.isPositiveNumber(value2))) {
+  if (!understandable(validator, value1, value2, 0, true)
+    && !(validator.isAnimationIterationCountKeyword(value2) || validator.isPositiveNumber(value2))) {
     return false;
-  } else if (validator.isVariable(value1) && validator.isVariable(value2)) {
+  } if (validator.isVariable(value1) && validator.isVariable(value2)) {
```

Comment:
The other thing I don't fancy is this 

```
if (...) {
  return 
} if (...) {
  return
}
```

construct. Any way to keep the `else`?

Reply:
@jakubpawlowicz I updated the config in [12620bb](https://github.com/clean-css/clean-css/pull/1201/commits/12620bb21853922a46919d406df8533dc1a95ec6):

```
    "no-else-return": ["error", { "allowElseIf": true } ],
```